src/library/traktor/traktorfeature.cpp

```diff
@@ -360,6 +361,15 @@ void TraktorFeature::parseTrack(QXmlStreamReader &xml, QSqlQuery &query) {
                 bpm = attr.value("BPM").toString().toFloat();
                 continue;
             }
+            if (xml.name() == QLatin1String("MUSICAL_KEY")) {
+                QXmlStreamAttributes attr = xml.attributes();
+                // Traktor happens to use the same key numbering
+                key = KeyUtils::keyToString(
+                        KeyUtils::keyFromNumericValue(
+                                attr.value("VALUE").toString().toInt()),
```

Comment:
```suggestion
                                attr.value("VALUE").toInt()),
```
avoids a heap-allocation for the temporary string.

Reply:
commited